packages/components/src/ui/create-styles/create-compiler/plugins/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import cssGridPlugin from 'styled-griddie';
+
+/**
+ * Internal dependencies
+ */
+import cssVariablesPlugin from './css-variables';
+import specificityPlugin from './extra-specificity';
```

Comment:
Can we add a comment to clarify that we can drop the plugins once IE11 is no longer supported?

Reply:
Yup! That's actually already called out in the README but I can move it to the top of the document to make it clearer.